test/src/helpers.cc

```diff
@@ -544,6 +544,18 @@ void get_supported_fs(
   tiledb_ctx_free(&ctx);
 }
 
+void get_supported_memfs(bool* memfs_supported) {
```

Comment:
Since this is a temporary bootstrapping, I added this as a separate function and didn't just extend "get_supported_fs" to avoid changing all the existing UT files that are calling this general function but are not testing memfs.

Reply:
Fine with me -- we'll revisit this when integrating with all of the other tests.